packages/block-library/src/query/edit/inspector-controls/enhanced-pagination-control.js

```diff
@@ -25,19 +25,19 @@ export default function EnhancedPaginationControl( {
 		);
 	} else if ( enhancedPagination ) {
 		help = __(
-			"Browsing between pages won't require a full page reload, unless non-compatible blocks are detected."
+			'Reload the full page—instead of just the posts list—when visitors navigate between pages.'
 		);
 	} else if ( hasUnsupportedBlocks ) {
 		help = __(
-			"Force page reload can't be disabled because there are non-compatible blocks inside the Query block."
+			"Enhancement disabled because there are non-compatible blocks inside the Query block."
```

Comment:
```suggestion
			'Enhancement disabled because there are non-compatible blocks inside the Query block.'
```

Reply:
Did you commit this or not? Thanks for the review!

Reply:
Yes, I did. See [ba35aa5](https://github.com/WordPress/gutenberg/pull/65081/commits/ba35aa50efbe605030cc93e4c21776856e5cbb08).